react-graphql-apollo-tutorial.md

```diff
@@ -819,18 +823,18 @@ const Link = ({ children, ...props }) => (
 export default Link;
 {{< /highlight >}}
 
-Once you start your application again, you should see a styled list of repositories with a name, url, description, the count of stars, owner and used language for the implementation of the project. If you cannot see any repositories, check whether your GitHub account has any public repositories in the first place. If it doesn't, then it's normal that nothing shows up. You can make yourself comfortable with GitHub by creating a couple of repositories for the sake of learning about GitHub but also to have this data available for this tutorial. Another way to create repositories for your own account is forking repositories from other people.
+Once you restart your application, you should see a styled list of repositories with a name, url, description, star count, owner, and the project's implementation language. If you can't see any repositories, check to see if your GitHub account has any public repositories. If it doesn't, then it's normal that nothing showed up. I reccomend you make yourself comfortable with GitHub by creating a couple of repositories, both for the sake of learning about GitHub and to use this data to practice with this tutorial. Another way to create repositories for your own account is forking repositories from other people.
```

Comment:
"I reccomend you make yourself comfortable" to "I recommend you make yourself comfortable"